lib/src/lib.rs

```diff
@@ -42,13 +41,29 @@ impl Logger {
     }
 }
 
+pub struct EntityResult {
```

Comment:
- See [this comment](https://github.com/FuelLabs/fuel-indexer/pull/65/files#r897328575)
- This feels....hacky :(